Cakefile

```diff
@@ -107,9 +107,9 @@ task "html", "Build HTML", (options) ->
   scripts = []
 
   if options.test
-    scripts.push "http://test.e17.dk/getnotaauthtoken"
+    scripts.push "http://nota.dk/getnotaauthtoken"
```

Comment:
Separate session and logins iirc. Server is also only available from the intranet.